### PR TITLE
oci: email server-crew-qa@l.c.c and OCI devels directly

### PR DESCRIPTION
**Pull request by paride**

### CODE REVIEW

File: jenkins-jobs/oci/integration.yaml [hunk 1]
```diff
@@ -42,4 +42,5 @@
           cd sts/lma-integration
           ./run-tests || exit 1
     publishers:
-      - email-squeaky-wheel
+      - email:
+          recipients: server-crew-qa@lists.canonical.com sergio.durigan@canonical.com
```
Comment (sergiodj) [inline jenkins-jobs/oci/integration.yaml:46]: I think it's worth copying @lucaskanashiro here as well.

File: jenkins-jobs/oci/unit.yaml [hunk 2]
```diff
@@ -57,4 +57,5 @@
           cd sts/oci-unit-tests
           DEBUG_TESTS=1 sh "{testcase}_test.sh" || exit 1
     publishers:
-      - email-squeaky-wheel
+      - email:
+          recipients: server-crew-qa@lists.canonical.com sergio.durigan@canonical.com
```
Comment (sergiodj) [inline jenkins-jobs/oci/unit.yaml:61]: Likewise.

**Review by sergiodj — CHANGES_REQUESTED** [review]

**Comment by paride** [comment]
Added @lucaskanashiro

**Comment by sergiodj** [comment]
Thanks, @paride.  @bryceharrington is also getting involved in the OCI work, but he's still learning the ropes so we can add him later.  I'm approving this PR.

**Pull request merged**